docs/src/pages/guides/batfish-helpers.md

```diff
@@ -25,6 +25,7 @@ These dataset functions often use the page's relative pathname as a unique ident
 - `siteBasePath`, required. The function requires the `siteBasePath`.
 - `data`, object. Provided by the data selector.
 - `sections`, array. See [Shape of multi-structured sections](#shape-of-multi-structured-sections).
+- `addPages`, array. See [Shape of appended pages](#shape-of-appended-pages).
```

Comment:
Could you add this to "Shape of multi-structured `sections`" as well? Around line 150.